Source/WebCore/svg/SVGFilterElement.cpp

```diff
@@ -67,23 +67,34 @@ void SVGFilterElement::attributeChanged(const QualifiedName& name, const AtomStr
 {
     SVGParsingError parseError = NoError;
 
-    if (name == SVGNames::filterUnitsAttr) {
+    switch (name.nodeName()) {
```

Comment:
Don’t we need to include `NodeName.h` here?

Reply:
Already included.